docs/resources/resources-github-copilot-prompt-library.md

```diff
@@ -0,0 +1,202 @@
+commands to get started in *Visual Studio Code*.
+```
+
+## Create an add-in project
+
+### Import a document as a Word template
+
+```code
+Create an Office JavaScript add-in for *Word* to *import a 
+document as a template*. List the steps to follow in *Visual 
+Studio Code* to create the add-in and insert code snippets in 
+the correct files.
+```
+
+### Create a custom function in Excel
+
+```code
+Create an Office JavaScript Add-in for *Excel* to *create a
+custom function in Excel*. List the steps to follow in *Visual 
+Studio Code* to create the add-in and insert code snippets in 
+the correct files.
+```
+
+### Insert graphics into a PowerPoint presentation slide
+
+```code
+Create an Office JavaScript Add-in for *PowerPoint* to *insert 
+graphics into a presentation slide*. List the steps to follow in 
+*Visual Studio Code* to create the add-in and insert code snippets 
+in the correct files.
+```
+
+## Implement a feature for Excel add-ins
+
+### Add a new worksheet
+
+```code
+Add a new worksheet *at the end* using the Excel JavaScript API.
+```
+
+### Get data from a table
+
+```code
+Retrieve the data in the range *A1:B3 on the first worksheet* 
+using the Excel JavaScript API.
+```
+
+### Create a chart
+
+```code
+Insert a line chart titled *"My chart"* into the current worksheet 
+using the data *in the range A1:B3* using the Excel JavaScript API.
+```
+
+### Create a custom function to conduct calculations in Excel
+
+```code
+Create a JavaScript custom function in Excel that conducts a 
+calculation.
+```
+
+### Handle an event
+
+```code
+Handle an event *when selection changes in worksheet* using 
+Excel JavaScript API.
+```
+
+### Create shapes
+
+```code
+Create *a yellow square* shape in the worksheet using the Excel 
+JavaScript API.
+```
+
+### Insert a copy of an existing workbook into the current one
+
+```code
+Insert a workbook template as base64 in the current workbook using 
+insertWorksheetsFromBase64 Excel JavaScript API.
+```
+
+## Implement a feature for Word add-ins
+
+### Insert a paragraph
+
+```code
+Insert a paragraph with the content *"My paragraph"* at the start 
+of the document using the Word JavaScript API.
+```
+
+### Apply a style to a paragraph
+
+```code
+Apply the style *"Heading1"* to the first paragraph in the document 
+using the Word JavaScript API.
+```
+
+### Change the font
+
+```code
+Change the font formatting of *the selected document text* using 
+the Word JavaScript API.
+```
+
+### Insert table into a document
+
+```code
+Insert a table named "Sample table" with sample data in the 
+document using the Word JavaScript API.
+```
+
+### Insert a content control
+
+```code
+Insert a content control labeled *"Sample Content Control"* on the 
+first paragraph and *set the content control's color to red* using 
+the Word JavaScript API.
+```
+
+### Add a comment
+
+```code
+Insert a comment with the content *"my comment"* on the document 
+selection in a Word add-in using Office.js.
```

Comment:
```suggestion
Insert a comment with the content *"my comment"* into the document 
selection with a Word add-in using Office.js.
```

Reply:
This is a tricky one. Seems that Copilot works best with **"Insert a comment into the document selection using the Word JavaScript API."**  When content in comment mentioned, it gets confused. So let's go with **this.**